packages/building-webpack/README.md

```diff
@@ -223,7 +223,11 @@ This the fastest method, as it strips away types during babel transformormation
 
 
 <details>
-  <summary>Supporting `LitElement`'s `@customElement` and `@property`</summary>
+  <summary>Supporting decorators</summary>
+
+::: warning
+Please note that our default recommendations do not support the use of decorators as adding non-native syntax to your code can be bad for maintenance longer term. If you do, however, want to add decorators, you can follow this section.
```

Comment:
We already have the same warning on top of this section. Do you think we should print it twice? Maybe we should restructure this section to differentiate between non-standard syntax and 'build jobs' like copy?

Reply:
I think if we're going to show people how to add non standard syntax to their code, we should add a disclaimer right there in that place that says we dont recommend it. Maybe we can make it smaller? Like:

```
::: warning
Please note that decorators will add [non standard syntax](https://open-wc.org/building/building-webpack.html#common-extensions) to your code.
```